src/pint/observatory/observatories.py

```diff
@@ -1,970 +1,87 @@
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_42ft",
-    aliases=["jb42"],
-    itrf_xyz=[3822294.825, -153862.275, 5085987.071],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_mkii",
-    aliases=["jbmk2"],
-    tempo_code="h",
-    itoa_code="J2",
-    itrf_xyz=[3822846.76, -153802.28, 5086285.9],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_mkii_rch",
-    aliases=["jbmk2roach"],
-    itrf_xyz=[3822846.76, -153802.28, 5086285.9],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_mkii_dfb",
-    aliases=["jbmk2dfb"],
-    itrf_xyz=[3822846.76, -153802.28, 5086285.9],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="lwa_sv",
-    aliases=["lwasv"],
-    itoa_code="LS",
-    itrf_xyz=[-1531155.54418, -5045324.30517, 3579583.8945],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="grao",
-    aliases=["grao"],
-    itrf_xyz=[6346273.531, -33779.7127, 634844.9454],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="srt",
-    aliases=["srt"],
-    tempo_code="z",
-    itoa_code="SR",
-    itrf_xyz=[4865182.766, 791922.689, 4035137.174],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-
-# From Tempo 2021 June 8
-TopoObs(
-    name="quabbin",
-    tempo_code="2",
-    itoa_code="QU",
-    itrf_xyz=[1430913.3496148302, -4495711.383965823, 4278113.974517222],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
-TopoObs(
-    name="vla_site",
-    tempo_code="c",
-    itoa_code="V2",
-    itrf_xyz=[-1601135.5133304405, -5042005.480977412, 3554875.076856462],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
-TopoObs(
-    name="gb_20m_xyz",
-    itoa_code="G2",
-    itrf_xyz=[883772.7974, -4924385.5975, 3944042.4991],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
-TopoObs(
-    name="northern_cross",
-    tempo_code="d",
-    itoa_code="BO",
-    itrf_xyz=[4461242.882451464, 919559.8351226494, 4449633.220012489],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
+import pint.config
+from pint.observatory.topo_obs import TopoObs
+from pint.observatory import bipm_default
+
+observatories_json = pint.config.runtimefile("observatories.json")
+
+pint_env_var = "PINT_OBS_OVERRIDE"
+
+__all__ = ["observatories_json", "read_observatories"]
+
+
+def get_default_value(entry, key, default=None):
+    """Return default value if key is not in entry.  Otherwise return entry[key]
+
+    Parameters
+    ----------
+    entry : dict
+    key : str
+    default :
+
+    Returns
+    -------
+        value for assignment
+    """
+
+    return default if not key in entry else entry[key]
+
+
+# Default values for instantiating a new TopoObs object
```

Comment:
Can you let the defaults in `TopoObs.__init__` handle this? Easiest is just not to pass unset values in `kwargs`.

Reply:
probably.  I sort of went about it backwards, but that should be fine.